docs/dotnet/dotnet-http-metrics.md

```diff
@@ -0,0 +1,257 @@
+# Semantic Conventions for HTTP-relevant metrics emitted by .NET and ASP.NET Core
```

Comment:
It seems like we have more metrics for outbound http, than inbound for asp.net core, and are adding enough dimensions that the metrics feel more like logging.

Reply:
can you elaborate your proposal? what we should/should not do here?